src/main/java/uk/gov/hmcts/reform/preapi/model/Participant.java

```diff
@@ -20,4 +20,14 @@ public class Participant {
     private Timestamp deletedAt;
     private Timestamp createdAt;
     private Timestamp modifiedAt;
+
+    public Participant(uk.gov.hmcts.reform.preapi.entities.Participant participantEntity) {
```

Comment:
better to import this to make the function definition neater?

Reply:
as above